src/styles/extensionStyles.sass

```diff
@@ -0,0 +1,91 @@
+#markdown-preview p
+    margin: 0
+
+textarea
+    height: 160px
+
+=titlediv
+    padding: 0px !important
+    text-align: left !important
+    margin: 8px 0px !important
+
+
+=mdpreview
+    color: #555
+    background-color: #FFF
+    margin: 0px
+    -webkit-appearance: none
+    overflow: auto
+    white-space: pre-wrap
+    word-wrap: break-word
+    height: 160px
+
+
+=mdpreview-bug
+    padding: 0px
+    font-family: 'Arial', 'Helvetica', sans-serif
+    color: #000
+    font-size: 13px
+    box-shadow: 0px 0px 3px rgba(0, 0, 0, .15)
+    border: 1px solid #B6B6B6
+
+
+.bugedit-lightning-mdpreview
+    +mdpreview
+    +mdpreview-bug
+
+
+.bugedit-lightning-titlediv
+    +titlediv
+
+
+.userstoryedit-lightning-mdpreview
+    +mdpreview
+    background-color: white
+    color: #16325C
+    border: 1px solid #D8DDE6
+    border-radius: 4px
+    font-weight: 300
+    padding: 12px
+    font-size: 18px
+
+
+.userstoryedit-lightning-titlediv
+    +titlediv
+
+
+.bugedit-classic-mdpreview
+    +mdpreview
+    +mdpreview-bug
+
+
+.bugedit-classic-titlediv
+    +titlediv
+
+
+.userstoryedit-classic-mdpreview
+    +mdpreview
+    font-family: Ubuntu, 'Lucida Grande', Arial, Helvetica, sans-serif
```

Comment:
remove `Arial, Helvetica,`.

* Helvetica looks bad in browsers on Windows.
* Arial looks bad in browsers on OSX.
* If you just put `sans-serif`, then Windows will default to Arial (which looks good on that platform), and OSX will default to Helvetica (which looks good on that platform).